twitter.css

```diff
@@ -0,0 +1,279 @@
+  padding-right: 16px;
+  margin: 0 auto;
+}
+
+.main-layout {
+  padding: 4px;
+}
+
+.main-layout > * {
+  padding: 2px;
+
+  border: 2px dashed lightblue;
+}
+
+.three-column-grid__expand-two {
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+}
+
+.three-row-grid__expand-two {
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+}
+
+.eight-columns {
+  display: grid;
+  grid-template-columns: repeat(8, 1fr);
+  grid-gap: 8px;
+}
+
+.two-column-expand-space{
+  display: grid;
+  grid-template-columns: 30px 30px auto;
+  margin: 0;
+}
+
+.main-header {
+  width: 275px;
+  height: 100vh;
+}
+
+.responsive-rows {
+  display: grid;
+  grid-auto-rows: repeat(auto-fit, minmax(800px, 1fr));
+}
+
+/* Layout */
+
+.resources-sidebar {
+  width: 275px;
+}
+
+.twitter-logo {
+  width: 55px;
+  height: 45px;
+  margin: 0;
+  padding: 10px;
+}
+
+.side-bar-menu {
+  font-weight: 800;
+  font-size: 20px;
+  margin-right: 20px;
+  padding: 20px;
+  cursor: pointer;
+}
+
+.side-bar-menu h2 {
+  font-weight: 800;
+  font-size: 20px;
+  margin-right: 20px;
+  padding: 20px;
+}
+
+.side-bar-menu :hover {
+  background-color: var(--twitter-background);
+  border-radius: 30px;
+  color: var(--twitter-color);
+  transition: color 100ms ease-out;
+}
+
+.side-bar-menu.active {
+  color: var(--twitter-color);
+}
+
+.side-bar-button__tweet {
+  width: 100%;
+  background-color: var(--twitter-color);
+  color: white;
+  border: none;
+  font-weight: 900;
+  border-radius: 30px;
+  margin-top: 20px;
+  height: 50px;
+}
+
+/* left side-bar */
+
+/* Feed Section */
+a {
+  text-decoration: none;
+
+}
+
+.scroll {
+  overflow-y: scroll;
+}
+
+.box-md {
+  height: 56px;
+}
+
+.main-box {
+  border: 2px solid var(--twitter-background);
+  margin: 3px;
+  min-height: 56px;
+  position: sticky;
+}
+
+.post-box {
+  border: 2px solid var(--twitter-background);
+  margin: 3px;
+  min-height: 300px;
+  overflow-y: scroll;
+}
+
+
+.menu-list {
+  list-style-type: none;
+  padding: 20px;
+  margin: 0;
+}
+
+.menu-list a {
+  font-weight: 800;
+  font-size: 20px;
+  margin-right: 20px;
+  /* padding: 20px; */
+  color: black;
+}
+
+.menu-list img {
+   font-weight: 800;
+   font-size: 20px;
+   margin-right: 10px;
+   padding: 30px;
+   padding-bottom: 0px;
+  }
+
+.active {
+  color: var(--twitter-color)
+}
+
+.menu-list  :hover {
+  /* background-color: var(--twitter-background);
+  border-radius: 40px; */
+  color: var(--twitter-color);
+  transition: color 100ms ease-out;
+
+}
+
+.feed-menu-icon {
+  width: 25px;
+  height: auto;
+}
+
+menu-list-link {
+
+}
+
+/* FEED */
+.profile-feed-pic img {
+  height: 40px;
+  border-radius: 50%;
+  object-fit: contain;
+  
+}
+
+span img {
+  height: 40px;
+  border-radius: 30px;
+  object-fit: contain;
+}
+
+/* POST */
+.post__badge {
+  font-size: 14px !important;
+  color: var(--twitter-color);
+  margin-right: 5px;
+}
+
+.post__header-Special {
+  font-weight: 600;
+  font-size: 12px;
+  color: gray;
+}
```

Comment:
Adding a greyish color to create a transparent like username to create a difference in tone from other text, worked well.